app/views/content_items/guide.html.erb

```diff
@@ -56,7 +56,7 @@
   <div class="govuk-grid-column-two-thirds" id="guide-contents">
     <% if @content_item.has_parts? %>
       <% if @content_item.show_guide_navigation? %>
-        <h1 class="part-title">
+        <h1 class="govuk-heading-l">
```

Comment:
Have you considered using the [heading component](https://components.publishing.service.gov.uk/component-guide/heading)?

Reply:
ah yeah, that would make sense!